### DISCUSSION TITLE
microphone

### DISCUSSION THREAD

**Discussion by pddpauw** [rdz-oss/BattyBirdNET-Pi#12] [Q&A]
Using birdnet-pi a couple of months now.
Was wondering if this bat analyzer would also work with other bat mic's.  I have a batbox (https://www.batbox.com/product/batbox-baton/) which can be connected to the PI (via USB sound card).  
Would this work, or which setting would have to be applied to make this work?

thx - Pieter

**Comment by rdz-oss** [comment]
Hi Pieter,

the microphone should be accpeted by the system if it complies to the usb microphone standart.

The isssue might be that the noise profile of your microphone is unknown to the system and hence that is likely to trigger more false ids from noise.

You can try and se if this is the case.

BR
Richard
